databases/business-sync/doc-definitions.js

```diff
@@ -33,17 +33,22 @@ function() {
 
   // Builds a function that returns the view, add, replace, remove channels extrapolated from the specified base privilege, name which is
   // formatted according to the de facto Books convention of "VIEW_FOOBAR", "ADD_FOOBAR", "CHANGE_FOOBAR" and "REMOVE_FOOBAR" assuming the
-  // base privilege name is "FOOBAR"
+  // base privilege name is "FOOBAR". If basePrivilegeName is omitted, only the write channel configuration is set (and only to the "STAFF"
+  // channel).
   function getDocSyncChannels(doc, oldDoc, basePrivilegeName) {
     var businessId = getBusinessId(doc, oldDoc);
 
     return function(doc, oldDoc) {
-      return {
-        view: [ toSyncChannel(businessId, 'VIEW_' + basePrivilegeName), staffChannel ],
-        add: [ toSyncChannel(businessId, 'ADD_' + basePrivilegeName), staffChannel ],
-        replace: [ toSyncChannel(businessId, 'CHANGE_' + basePrivilegeName), staffChannel ],
-        remove: [ toSyncChannel(businessId, 'REMOVE_' + basePrivilegeName), staffChannel ]
-      };
+      if (basePrivilegeName) {
+        return {
+          view: [ toSyncChannel(businessId, 'VIEW_' + basePrivilegeName), staffChannel ],
+          add: [ toSyncChannel(businessId, 'ADD_' + basePrivilegeName), staffChannel ],
+          replace: [ toSyncChannel(businessId, 'CHANGE_' + basePrivilegeName), staffChannel ],
+          remove: [ toSyncChannel(businessId, 'REMOVE_' + basePrivilegeName), staffChannel ]
+        };
+      } else {
+        return { write: staffChannel };
```

Comment:
Indeed, under the `channels` property:  https://github.com/Kashoo/synctos#essential-document-properties